website/docs/components/badge-count/partials/code/showcase.md

```diff
@@ -1,7 +1,7 @@
 <section data-test-percy data-section="showcase">
-  
+  <h2>Showcase</h2>
```

Comment:
Updating to use markdown headings so that it appears in the sidecar.
```suggestion
## Showcase
```

Reply:
If you update the rest of the headings in this section they will also appear in the sidecar :)

Reply:
So we can't just convert it to markdown, there are other things to do (including removing the wrapping section). It also won't convert successfully unless it's _all_ converted. I think that's why we decided in today's sync to leave the showcase sections alone for the time being.

Reply:
What about removing `<h2>Showcase</h2>` and putting `## Showcase` above the section and leaving the rest as is? That way the section is still accessible via the sidecar.

Reply:
Thanks, I tried converting but got errors so I mostly left alone as recommended in the guidelines for editing content: https://github.com/hashicorp/design-system/blob/main/wiki/Website-Editing-existing-content.md

Reply:
Oops, missed your last reply @heatherlarsen I just tried it and it worked so I'll make this update and include as part of the next component docs I work on.